src/pQueue.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+export default class pQueue {
+  constructor() {
+    this.storage = []
+    this.top = 0
+  }
+
+  enqueue(name, priority) {
+    this.storage[this.top++] = {name, priority}
+    return this.storage
+  }
+
+  isEmpty() {
+    return this.top === 0
+  }
+
+  length() {
+    return this.top
```

Comment:
I agree with @shakalee14 comment here; in the context of a `PriorityQueue`, the name `front` would more clearly reveal the intention of this variable.